stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/util/AddressUtil.java

```diff
@@ -0,0 +1,80 @@
+package com.bloxbean.cardano.yaci.store.governance.util;
+
+import com.bloxbean.cardano.client.address.Address;
+import com.bloxbean.cardano.client.address.AddressType;
+import com.bloxbean.cardano.client.common.model.Network;
+import com.bloxbean.cardano.client.common.model.Networks;
+import com.bloxbean.cardano.client.exception.AddressRuntimeException;
+import com.bloxbean.cardano.client.spec.NetworkId;
+import com.bloxbean.cardano.client.util.HexUtil;
+import com.bloxbean.cardano.yaci.core.model.certs.StakeCredential;
+import lombok.NonNull;
+
+import static com.bloxbean.cardano.client.address.util.AddressEncoderDecoderUtil.*;
+
+//TODO -- Remove this class once the cardano-client-lib is updated
+public class AddressUtil {
+    private static final byte RWD_STAKE_KEY_HEADER_TYPE = (byte)0b1110_0000;
+    private static final byte RWD_STAKE_SCRIPT_HEADER_TYPE = (byte)0b1111_0000;
+
+    public static Address getRewardAddress(@NonNull StakeCredential stakeCredential, boolean isMainnet) {
```

Comment:
This method is now there in CCL 0.5.0.  I think we don't need this class any more.

```
Address getRewardAddress(@NonNull Credential stakeCredential, Network networkInfo)
```